admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -107,7 +107,7 @@ export const ApplicationPage: React.FunctionComponent = () => {
     const createEventType = () => {
         setShowModal(true);
         setIsEdit(false);
-        setEventTypes({});
+        setEventTypes({visible:true});
```

Comment:
What is the effect of that line?

Reply:
It defines the event type default value on creation flow.

Reply:
So `visible` will be true by default and the other boolean values will be false?